src/ragger/navigator/navigation_scenario.py

```diff
@@ -31,7 +31,10 @@ def __init__(self, device: str, use_case: UseCase, approve: bool):
                 self.pattern = "^Confirm$"
 
             elif use_case == UseCase.TX_REVIEW:
-                self.navigation = NavInsID.USE_CASE_REVIEW_TAP
+                if device.startswith("stax"):
+                    self.navigation = NavInsID.USE_CASE_REVIEW_TAP
```

Comment:
No SWIPE for `ADDRESS_CONFIRMATION` above?

Reply:
I'm not aware of all the cases where the SWIPE is mandatory; I get from your comment that it is for `ADDRESS_CONFIRMATION` so I'll add it.

Reply:
I'd say that swipe is the flex equivalent of tap on stax :)

Reply:
Okay, that will be simpler then!